libcxx/utils/libcxx/test/params.py

```diff
@@ -165,11 +165,28 @@ def getSuitableClangTidy(cfg):
         default=lambda cfg: next(
             s for s in reversed(_allStandards) if getStdFlag(cfg, s)
         ),
-        actions=lambda std: [
+        actions=lambda std: filter(None, [
```

Comment:
I think we should add Lit support for arithmetic expressions instead. @jdenny-ornl do you have thoughts on this?

Reply:
I prefer this PR's approach.  It keeps lit's implementation simple and defers to individual test suites to specify how they need sorting to be done.

@mordante In the initial PR comment, would you please add a link to your discourse for the sake of people arriving here?